docs/reference/profiles.md

```diff
@@ -118,15 +118,18 @@ For example, a mixture of taints and affinity can put less critical functions on
 In this example, we create a Profile using taints and affinity to place functions on the node with a GPU.  We will also ensure that _only_ functions that require the GPU are scheduled on these nodes. This ensures that the functions that need to use the GPU are not blocked by other standard functions taking resources on these special nodes.
 
 1. Install the latest `faas-netes` release and the CRD. The is most easily done with [`arkade`](https://github.com/alexellis/arkade)
+
     ```sh
     arkade install openfaas
     ```
+
     This default installation will enable Profiles.
 
 2. Label _and_ Taint the node with the GPU
+
     ```sh
-    kubectl labels nodes node1 gpu=installed
-    kubectl taint nodes node1 gpu:NoSchedule
+    kubectl label nodes <NODE_NAME> gpu=installed
+    kubectl taint node <NODE_NAME> gpu:NoSchedule
```

Comment:
```suggestion
    kubectl taint nodes <NODE_NAME> gpu:NoSchedule
```

the help text for `kubectl taint --help` uses the plural here although i suspect both forms are supported. But using the plural on both lines will make it more consistent.

Reply:
If you want to accept this @cpanato, you'll need to manually type in your Signed-off-by line in the commit message.

<img width="924" alt="Screenshot 2021-10-31 at 09 52 47" src="https://user-images.githubusercontent.com/6358735/139576930-5e6e2be6-0956-4d92-9abe-e36f73d84d6f.png">

Reply:
thanks for the review and suggestions, made the requested change